services/skus/handler/handler.go

```diff
@@ -0,0 +1,81 @@
+package handler
+
+import (
+	"context"
+	"errors"
+	"net/http"
+
+	"github.com/asaskevich/govalidator"
+
+	"github.com/brave-intl/bat-go/libs/handlers"
+	"github.com/brave-intl/bat-go/libs/logging"
+	"github.com/brave-intl/bat-go/libs/requestutils"
+
+	"github.com/brave-intl/bat-go/services/skus/model"
+)
+
+type orderService interface {
+	CreateOrderFromRequest(ctx context.Context, req CreateOrderRequest) (*model.Order, error)
+}
+
+type Order struct {
+	svc orderService
+}
+
+func NewOrder(svc orderService) *Order {
+	result := &Order{
+		svc: svc,
+	}
+
+	return result
+}
+
+func (h *Order) Create(w http.ResponseWriter, r *http.Request) *handlers.AppError {
+	ctx := r.Context()
+
+	var req CreateOrderRequest
+	if err := requestutils.ReadJSON(r.Context(), r.Body, &req); err != nil {
+		return handlers.WrapError(err, "Error in request body", http.StatusBadRequest)
+	}
+
+	if _, err := govalidator.ValidateStruct(req); err != nil {
+		return handlers.WrapValidationError(err)
+	}
+
+	if len(req.Items) == 0 {
+		return handlers.ValidationError(
+			"Error validating request body",
+			map[string]interface{}{
+				"items": "array must contain at least one item",
+			},
+		)
+	}
+
+	lg := logging.Logger(ctx, "payments").With().Str("func", "CreateOrderHandler").Logger()
```

Comment:
Decided to keep these values for now to avoid anything potentially dependent on it.